pkgs/os-specific/linux/game-devices-udev-rules/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+}:
+stdenv.mkDerivation rec {
+  pname = "game-devices-udev-rules";
+  version = "0.21";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "fabiscafe";
+    repo = "game-devices-udev";
+    rev = version;
+    hash = "sha256-Yy91yDF5BSDTTlr/Pj8e0UklPooEdzvRW8mkhdHtHVo=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+  install -Dm444 -t "$out/lib/udev/rules.d" *.rules
```

Comment:
The indentation is a bit off. Also please squash the commits.

Reply:
Fixed. Thought I squashed them, but it's not showing. I'll see if I can figure that out.